### PR TITLE
[AD-271] Documentation: Clarify unbundled certificate should be specified in a connection properties

### PR DESCRIPTION
**Pull request by birschick-bq**
### Summary

[AD-271] [Documentation] Clarify unbundled certificate should be specified in a connection properties

### Description

Added Troubleshooting Guide and updated indices.

NOTE: Branched for existing [PR](https://github.com/aws/amazon-documentdb-jdbc-driver/pull/27).

### Related Issue

https://bitquill.atlassian.net/browse/AD-271

### Additional Reviewers
@birschick-bq
@mitchell-elholm
@alexey-temnikov 
@andiem-bq

### CODE REVIEW

File: src/markdown/support/trouble-shooting.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Troubleshooting Guide
```
Comment (andiemontoyeah) [inline src/markdown/support/trouble-shooting.md]: change file name to trouble-shooting.md -> troubleshooting.md ?

File: src/markdown/sql/sql-jdbc-limitations.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# SQL and JDBC Support and Limitations
```
Comment (alexey-temnikov) [inline src/markdown/sql/sql-jdbc-limitations.md]: Suggesting to expand this section, to add all current limitations for the driver. See example how it is done by Redshift team (https://docs.aws.amazon.com/redshift/latest/dg/c_redshift-and-postgres-sql.html). Suggesting to raise task so we can track it.

Comment (birschick-bq) [reply src/markdown/sql/sql-jdbc-limitations.md]: Created JIRA task : https://bitquill.atlassian.net/browse/AD-302 to track this improvement to documentation.

File: src/markdown/support/trouble-shooting.md (hunk 1 above)
Comment (birschick-bq) [reply src/markdown/support/trouble-shooting.md]: Fixed.

**Review by andiemontoyeah — APPROVED** [review]

**Pull request merged**